CRM/Case/Form/CaseView.php

```diff
@@ -425,7 +425,7 @@ public function postProcess() {
     $session = CRM_Core_Session::singleton();
     $session->pushUserContext($url);
 
-    if (!empty($params['timeline_id']) && !empty($_POST['_qf_CaseView_next'])) {
+    if (!empty($params['timeline_id']) && $buttonName = '_qf_CaseView_next') {
```

Comment:
@demeritcowboy missing a 2nd = here

Reply:
It's there I just used a really tiny font. No seriously yikes good catch.

Oh also should this be against 5.33 or are you branching real soon?

Reply:
we should be branching soon and I would keep it against master for the moment

Reply:
ok thanks